app/agents/tools.py

```diff
@@ -4,7 +4,7 @@
 from langchain.base_language import BaseLanguageModel
 
 from .tp_tools import *
-from .s2 import SemanticSearch
```

Comment:
@combspk , you are removing changes from my last PR. Could you keep this  line in? Thanks!

Reply:
Sorry, should be resolved now, forgot to pull first